src/Api/Api.php

```diff
@@ -227,4 +227,17 @@ public function getServiceSchema(
     {
         return $this->mapping->find($name, $version);
     }
+
+    /**
+     * Disabled.
+     *
+     * This SDK implementation has no async support.
+     *
+     * @see https://github.com/kusanagi/katana-sdk-spec#apidone--boolean
+     * @return bool
+     */
+    public function done(): bool
+    {
+        return false;
```

Comment:
Wouldn't this throw an error?

Reply:
Yes, it was pending the merge to the spec